infra/modules/service/dns.tf

```diff
@@ -0,0 +1,52 @@
+#
+# DNS names using Amazon Route 53 and SSL certificates using ACM.
```

Comment:
i've been on the fence with this but for consistency i've gone with the approach of putting everything in `main.tf` rather than splitting resources across multiple files, and using comments to separate out logical sections